sqlglot/optimizer/qualify_tables.py

```diff
@@ -66,13 +78,20 @@ def qualify_tables(
                     if not source.args.get("catalog"):
                         source.set("catalog", exp.to_identifier(catalog))
 
+                # Unnest parenthesized table
+                if isinstance(source.parent, exp.Paren):
+                    source.parent.replace(source)
+
+                # Unnest joins attached in tables by appending them to closest query
+                for join in source.args.get("joins") or []:
+                    scope.expression.append("joins", join)
+                else:
```

Comment:
you don't need an else here, it doesn't change anything

Reply:
Thanks, got rid of it.